### ISSUE TITLE
CBA_fnc_addBinocularMagazine breaks inventory sync

### ISSUE THREAD

**Issue by PabstMirror** [CBATeam/CBA_A3#347]
`setUnitLoadout` seems to break item/magazine sync in 1.60 on dedicated server.

Before `getUnitLoadout`:
```
[[],[],[],["U_C_Poloshirt_stripped",[["ACE_microDAGR",3]]],["V_PlateCarrierSpec_rgr",[]],["B_Bergen_blk",[]],"H_HelmetB_light_desert","",[],["","","","","",""]]
```
Both client and server see same loadout. The call:
```
player addWeapon "Laserdesignator"
[player, "Laserbatteries"] call CBA_fnc_addBinocularMagazine;
```

Now server's `getUnitLoadout` is missing the microDagrs
```
[[],[],[],["U_C_Poloshirt_stripped",[]],["V_PlateCarrierSpec_rgr",[]],["B_Bergen_blk",[]],"H_HelmetB_light_desert","",["Laserdesignator","","","",["Laserbatteries",1],[],""],["","","","","",""]]
```

Same results with comparing `items`
`[]` on server, `["ACE_microDAGR","ACE_microDAGR","ACE_microDAGR"]` on player

https://feedback.bistudio.com/T116980

**Comment by commy2** [comment]
I don't think we can do anything about this. There is no other way to add a binocular magazine reliably and I'm pretty sure BI will fix this on their own. Close?

**Comment by Killswitch00** [comment]
Yes. What we can do is add it to the list of known issues (if it isn't there already).

**Comment by PabstMirror** [comment]
Command seems to be fixed in 1.62

I still have serious problems with CBA adding this function as it can break compatibility with many other addons.

It will reset all unique magazineIDs (will break ACE_dogtags)

It will also reset variables set on player's uniform and backpack (I believe this will break TFAR radio backpacks)

```
(backpackContainer player) setVariable ["x", 2]
[player, "Laserbatteries"] call CBA_fnc_addBinocularMagazine;
(backpackContainer player) getVariable "x" //Undefined
```
